dataflux_core/tests/fake_gcs.py

```diff
@@ -102,25 +110,12 @@ def download_as_bytes(self, retry=None):
 
     def open(self, mode: str, ignore_flush: bool = False):
         if mode == "rb":
-            return BlobReader()
+            return io.BytesIO(self.content)
         elif mode == "wb":
-            return BlobWriter()
-
+            self.content = b''
+            return FakeBlobWriter(self)
         return None
```

Comment:
Would it make sense to raise a not-implemented exception here instead of returning None?